src/applications/personalization/dashboard/tests/e2e/benefit-payments-and-debt.cypress.spec.js

```diff
@@ -0,0 +1,120 @@
+import { generateFeatureToggles } from 'applications/check-in/api/local-mock-api/mocks/v2/feature-toggles';

```

Comment:
To limit cross-application connections, you should create your own `generateFeatureToggles`

Reply:
@mdewey @timwright12 we should probably add a note about this to the documentation about feature toggles so it is more clear to new engineers, thoughts?

Reply:
@theodur is creating a script to check for things like this, we can have it run on changed files in a precommit hook